kroxylicious/src/main/resources/log4j2.properties

```diff
@@ -11,11 +11,23 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} <%t> %-5p %c:%L - %m%n
 
-rootLogger.level = WARN
+rootLogger.level = ${env:KROXYLICIOUS_ROOT_LOG_LEVEL:-WARN}
 rootLogger.appenderRefs = stdout
 rootLogger.appenderRef.console.ref = STDOUT
 rootLogger.additivity = false
 
-#logger.netty.name = io.netty
-#logger.netty.level = TRACE
-#logger.netty.additivity = false
+# The DownstreamNetworkLogger and DownstreamFrameLogger loggers are required to allow
+# the virtual cluster logNetwork/logFrames feature to work.  The log level needs to be in sync
+# with the logger level used in the code.
+logger.DownstreamNetworkLogger.name = io.kroxylicious.proxy.internal.DownstreamNetworkLogger
+logger.DownstreamNetworkLogger.level = INFO
+
+logger.DownstreamFrameLogger.name = io.kroxylicious.proxy.internal.DownstreamFrameLogger
+logger.DownstreamFrameLogger.level = INFO
+
+logger.StartupShutdownLogger.name = io.kroxylicious.proxy.internal.StartupShutdownLogger
+logger.StartupShutdownLogger.level = INFO
+
+logger.test.name = io.kroxylicious.proxy.KafkaProxy
+logger.test.level = ${env:KROXYLICIOUS_LOG_LEVEL:-${env:KROXYLICIOUS_ROOT_LOG_LEVEL:-WARN}}
```

Comment:
Thinking out loud should *tests* default to more verbose logging? Are we more likely to catch issues etc when running tests with logging turned up? 

A quiet build is also a desirable property. I'm tempted to say the build server should run with a higher level of logging on by default so we can diagnose issues there (which this change facilitates nicely)?